app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from os import makedirs, remove, path, rename
+
+
+def move_file(command: str) -> None:
+    source_data = command.split()
+    paths = source_data[2].split("/")
+    new_path = "/".join(paths[:-1])
+    if "/" in source_data[2]:
+        makedirs(new_path)
+        with open(source_data[1], "w+") as file_in, \
+             open(path.join(new_path, paths[-1]), "w") as file_out:
+            save_data = file_in.read()
+            file_out.write(save_data)
+        remove(source_data[1])
+    else:
+        with open(source_data[1], "w+") as file_in:
+            pass
```

Comment:
what a sense of this?

Reply:
I can't rename file if it not exist

Reply:
Maybe task for renaming not clear for me

Reply:
yes, but you can it more simply, don't use empty context managers

Reply:
just open(source_data[1], "w+") ?

Reply:
try to find method for rename files, if this file doesn't exist, raise an error